xinference/core/restful_api.py

```diff
@@ -266,6 +277,20 @@ def serve(self):
             methods=["POST"],
             response_model=ChatCompletion,
         )
+
+        # for custom models
+        self._router.add_api_route(
+            "/v2/models/register_model", self.register_model, methods=["POST"]
```

Comment:
Adding new functionality won't break backward compatiblity. Thus, it is safe to use `v1` for custom model APIs.

Reply:
udpated in the latest commit